src/Translations/it/microtf2.phrases.txt

```diff
@@ -1,5 +1,18 @@
 "Phrases"
 {
+	"Startup_Header"
+	{
+		"it"		"Benvenuto a WarioWare!"
+	}
+	"Startup_BodyIntro"
+	{
+		"it"		"Il gioco comincerà a breve, sii pronto!"
+	}
+	"Startup_BodyCountdown"
+	{
+		"#format"	"{1:i}"
```

Comment:
Can you remove this line? This line is only needed in the main translations file